server/build_event_protocol/target_tracker/target_tracker.go

```diff
@@ -154,7 +154,7 @@ func isTest(t *target) bool {
 	return strings.HasSuffix(strings.ToLower(t.ruleType), "test")
 }
 
-func (t *TargetTracker) testTargetsInAtleastState(state targetState) bool {
+func (t *TargetTracker) testTargetsInAtLeastState(state targetState) bool {
```

Comment:
lol i had to read this like 10 times to see the delta. At least I finally got it.

Reply:
itsAsubtleOne